AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_LightAdded.py

```diff
@@ -97,74 +182,299 @@ def AtomEditorComponents_Light_AddedToEntity():
+                #         f"P1: Shadowmap size for Light type: {light_type} failed to be set to {shadowmap_size}.")
+                #     Report.result(test_shadowmap_size, current_shadowmap_size == SHADOWMAP_SIZE[shadowmap_size])
+
+                # Shadow filter method tests.
+                # 17. Set the Shadow filter method.
+                for filter_method in SHADOW_FILTER_METHOD.keys():
+                    light_component.set_component_property_value(
+                        AtomComponentProperties.light('Shadow filter method'), SHADOW_FILTER_METHOD[filter_method])
+                    current_filter_method = light_component.get_component_property_value(
+                        AtomComponentProperties.light('Shadow filter method'))
+                    test_shadow_filter_method = (
+                        f"Shadow filter method set to {filter_method}",
+                        f"P1: Shadow filter method set to {filter_method}")
+                    Report.result(test_shadow_filter_method,
+                                  current_filter_method == SHADOW_FILTER_METHOD[filter_method])
+
+                    # 18. Edit the Filtering sample count parameter.
+                    if current_filter_method in (1, 3):
+                        light_component.set_component_property_value(
+                            AtomComponentProperties.light('Filtering sample count'), 64)
+                        Report.result(Tests.edit_filtering_sample_count,
+                                      light_component.get_component_property_value(
+                                          AtomComponentProperties.light('Filtering sample count')) == 64)
+
+                    # 19. Edit the ESM Exponent parameter.
+                    if current_filter_method in (2, 3):
+                        light_component.set_component_property_value(
+                            AtomComponentProperties.light('ESM exponent'), 5000)
+                        Report.result(Tests.edit_esm_exponent,
+                                      light_component.get_component_property_value(
+                                          AtomComponentProperties.light('ESM exponent')) == 5000)
+
+                # 20. Disable Shadows (re-enabled after test for game mode verification):
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shadow'), False)
+                general.idle_wait_frames(1)
+                Report.result(
+                    Tests.disable_shadows,
+                    light_component.get_component_property_value(
+                        AtomComponentProperties.light('Enable shadow')) is False)
+                light_component.set_component_property_value(
+                    AtomComponentProperties.light('Enable shadow'), True)
+
+            # Shutter tests for applicable light types:
+            if current_light_type in (2, 7):
```

Comment:
instead of using numbers like `2` and `7` for these calls can we use the `LIGHT_TYPE` dict reference for the number instead? it makes it much cleaner to read which light type its referring to without having to check `atom_constants.py`

Reply:
I went with using the keys from the LIGHT_TYPE dictionary:
if light_type in ('spot_disk', 'simple_spot'):